solvers/PGD.py

```diff
@@ -24,25 +26,36 @@ class Solver(BaseSolver):
         "kolmogorov"
     ]}
 
-    def set_objective(self, lin_op, reg, y, isotropy):
-        self.reg = reg
-        self.isotropy = isotropy
-        self.lin_op, self.y = lin_op, y
-
-    def skip(self, lin_op, reg, y, isotropy):
+    def skip(self, lin_op, reg, delta, data_fit, y, isotropy):
         if isotropy != "anisotropic":
             return True, "prox-tv supports only anisoTV"
         return False, None
 
+    def set_objective(self, lin_op, reg, delta, data_fit, y, isotropy):
+        self.reg, self.delta = reg, delta
+        self.isotropy = isotropy
+        self.data_fit = data_fit
+        self.lin_op, self.y = lin_op, y
```

Comment:
I see it called lin_op here. You should use consistent naming across benchmarks. See how it's called in your TV1D benchmark @EnLAI111